public/views/create.html

```diff
@@ -61,6 +61,7 @@
             </label>
             <label for="ext-master" class="m10t">
               <span translate>Master extended private key</span>
+              <small translate>If not given, a random key will be generated</small>
```

Comment:
Yes. The template.pot files is out-of-date. I think we 're going to add more words in the following of days and updating the pot file. I know, it 's a hard work to maintenance, but we are currently changing words almost every days to do that all users understand Copay better.